Content.Shared/Pinpointer/SharedPinpointerSystem.cs

```diff
@@ -27,12 +30,31 @@ private void OnAfterInteract(EntityUid uid, PinpointerComponent component, After
         if (!args.CanReach || args.Target is not { } target)
             return;
 
-        if (!component.CanRetarget || component.IsActive)
+        if (component.IsActive && component.CanRetarget)
             return;
 
         // TODO add doafter once the freeze is lifted
         args.Handled = true;
-        component.Target = args.Target;
+
+        if (component.CanRetarget)
+        {
+            component.Target = args.Target;
+        }
+
+        else if (TryComp<NukeCodePaperComponent>(args.Target, out var nukeCodePaperComponent))
```

Comment:
this emagged behavior seems very weird the way it doesn't track the nuke based on the paper and instead tracks the paper. i'm not really save what would be better though.

Reply:
Maybe just have the emaggied Pointer also track multiple targets and have a way to clear it like trying to track itself?

Reply:
I made it so it will track the paper if it's turned off(it's behavior before this PR) and will track the nuke if it scans the paper while turned on.